readme.md

```diff
@@ -615,6 +615,27 @@ The order where these resources are used are:
 
 Here's the full list of supported [configuration options](config.md).
 
+###### Examples
```

Comment:
Remarks:
- I'm a bit confused about the location of that documentation example defined with this PR as it seems to appear in the middle of the doc instead to be a well defined section of Halkyon Component CRD for Annotationless
- Instead of writing `Note that some items does not end up in the same hierarchical place where they were configured.`, I would suggest to explain instead : 
  - how a user can use the annotationless mode 
  - how I can override